### PR TITLE
Replication backlog [doc-680]

### PR DESCRIPTION
**Pull request by AlonMagrafta**
@lanceleonard @kaitlynmichael please review my english, wording etc :) I assume we will iterate (a bit) on it

### CODE REVIEW

File: content/rs/administering/database-operations/updating-configurations.md [hunk 1]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
 If a participating cluster that was removed forcefully returns attempts to re-join the cluster,
 it will have an out of date on Active-Active database membership.
 The joined participating clusters reject updates sent from the removed participating cluster.
+
+## Replication Backlog
+
+Active-Active and Active-Passive databases are maintaining a Replication Backlog to keep synchronization among the clusters.
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:65]:
```suggestion
Active-Active and Active-Passive databases maintain a Replication Backlog to synchronize clusters.
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 2]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+By default, the Replication Backlog size will be set automatically allocated to 1% of the database size and be limited in the range of 1MB (min) - 250MB (max) per shard.
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:66]:
```suggestion
By default, the replication backlog is set to one percent (1%) of the database size and ranges between 1MB to 250MB per shard.
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 3]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+It is also possible to toggle between the automatic allocation to a manual size allocation from the command-line:
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:67]:
```suggestion
Use the [crdb-cli]({{< relref "rs/references/crdb-cli-reference.md" >}}) utility to control the size of the replication backlog.  You can set it to `auto` or set a specific size.  

The syntax varies between Active-Active and Active-Passive databases. 
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 4]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+
+For an Active-Active database use [crdb-cli]({{< relref "rs/references/crdb-cli-reference.md" >}}) to run:
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:69]:
```suggestion
For an Active-Active database:
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 5]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+```text
+crdb-cli crdb update --crdb-guid <crdb_guid> --default-db-config "{\"crdt_repl_backlog_size\": <SIZE_IN_BYTES | 'auto'>}"
+```
+
+For an Active-Passive database use [rladmin]({{< relref "rs/references/rladmin.md" >}}) from the source database and run:
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:74]:
```suggestion
For an Active-Passive database:
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 6]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+```text
+rladmin tune db <db:id | name> repl_backlog <Backlog size in MB or 'auto'>
+```
+
+**For Redis Software before version 6.0.20:**
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:79]:
```suggestion
**For Redis Software versions earlier than 6.0.20:**
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 7]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+Replication backlog is set by default to 1MB and is can not be set dynamically ('auto' mode).
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:80]:
```suggestion
The Replication backlog defaults to 1MB and cannot be set dynamically ('auto' mode).
```

File: content/rs/administering/database-operations/updating-configurations.md [hunk 8]
```diff
@@ -59,3 +59,27 @@ If you must remove offline participating clusters, you can do this with forced r
...
+Tuning is possible only using [rladmin]({{< relref "rs/references/rladmin.md" >}}) applying to a local intanse of the database on each cluster.
```
Comment (lanceleonard) [inline content/rs/administering/database-operations/updating-configurations.md:81]:
```suggestion
To control the size of the replication log, use [rladmin]({{< relref "rs/references/rladmin.md" >}}) to tune the local database instance in each cluster.
```

**Review by lanceleonard — CHANGES_REQUESTED** [review]
Should we also add info describing where to find the replication log?

File: content/rs/administering/database-operations/updating-configurations.md [hunk 9]
```diff
@@ -59,3 +59,28 @@ If you must remove offline participating clusters, you can do this with forced r
 If a participating cluster that was removed forcefully returns attempts to re-join the cluster,
 it will have an out of date on Active-Active database membership.
 The joined participating clusters reject updates sent from the removed participating cluster.
+
+## Replication Backlog
+
+Active-Active and Active-Passive databases maintain a Replication Backlog to synchronize clusters.
```
Comment (kaitlynmichael) [inline content/rs/administering/database-operations/updating-configurations.md:65]:
```suggestion
Active-Active and Active-Passive databases maintain a replication backlog to synchronize clusters.
```
I don't believe these need to be capitalized

File: content/rs/administering/database-operations/updating-configurations.md [hunk 10]
```diff
@@ -59,3 +59,28 @@ If you must remove offline participating clusters, you can do this with forced r
 If a participating cluster that was removed forcefully returns attempts to re-join the cluster,
 it will have an out of date on Active-Active database membership.
 The joined participating clusters reject updates sent from the removed participating cluster.
+
+## Replication Backlog
```
Comment (kaitlynmichael) [inline content/rs/administering/database-operations/updating-configurations.md:63]:
```suggestion
## Replication backlog
```
We use sentence case

File: content/rs/administering/database-operations/updating-configurations.md [hunk 11]
```diff
@@ -59,3 +59,28 @@ If you must remove offline participating clusters, you can do this with forced r
...
+crdb-cli crdb update --crdb-guid <crdb_guid> --default-db-config "{\"crdt_repl_backlog_size\": <SIZE_IN_BYTES | 'auto'>}"
+```
+
+For an Active-Passive database:
+```text
+rladmin tune db <db:id | name> repl_backlog <Backlog size in MB or 'auto'>
+```
+
+**For Redis Software versions earlier than 6.0.20:**
+The Replication backlog defaults to 1MB and cannot be set dynamically ('auto' mode).
```
Comment (kaitlynmichael) [inline content/rs/administering/database-operations/updating-configurations.md:82]:
```suggestion
The replication backlog defaults to 1MB and cannot be set dynamically with 'auto' mode.
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]

**Review by kaitlynmichael — APPROVED** [review]

**Review by lanceleonard — APPROVED** [review]
LGTM

**Comment by AlonMagrafta** [comment]
@kaitlynmichael @lanceleonard 
I figured out I need to modify things here. Please don't merge to latest yet. Thanks.

**Pull request merged**